website/src/components/Home.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import Dashboard from './dashboard/Dashboard';
+import Landing from './landing/Landing';
+
+class Home extends React.Component {
+   /**
+    * Renders HTML for the home page depending on whether user is
+    * logged in or not.
+    * @returns {HTML}    The HTML of the home page.
+    */
+   render() {
+     const userId = this.props.userInfo.userId;
```

Comment:
Ok, sounds good. I think there was a reason I was avoiding firebase.auth().currentUser, but is it ok if I do it directly in the file? Or should I make a const in LoginHelper?